primitives/src/pcs/univariate_kzg/mod.rs

```diff
@@ -655,18 +655,12 @@ where
 pub(crate) mod icicle {
     use super::*;
     use crate::icicle_deps::{curves::*, *};
-    use ark_ec::{short_weierstrass::Affine, CurveConfig};
-    use ark_ff::BigInteger;
 
     /// Trait for GPU-accelerated PCS.commit APIs
-    pub trait GPUCommit<E, C>
-    where
-        C: IcicleCurve + MSM<C>,
-        C::ScalarField: ArkConvertible<ArkEquivalent = E::ScalarField>,
-        C::BaseField: ArkConvertible<ArkEquivalent = <C::ArkSWConfig as CurveConfig>::BaseField>,
-        <C::ArkSWConfig as CurveConfig>::BaseField: PrimeField,
-        E: Pairing<G1Affine = Affine<<C as IcicleCurve>::ArkSWConfig>>,
-    {
+    pub trait GPUCommit<E: Pairing> {
```

Comment:
nit: maybe call it `GPUCommittable`

Reply:
done in [b471f28](https://github.com/EspressoSystems/jellyfish/pull/515/commits/b471f28514bf59f905900f355c09372e4a74ac6c)